### DISCUSSION TITLE
actinia at FOSS4G 2023

### DISCUSSION THREAD

**Discussion by ninsbl** [mundialis/actinia_core#418] [General]
Hi,

The program for FOSS4G 2023 (26 June - 2 July 2023 in Prizren/Kosovo, https://2023.foss4g.org/) is unfortunately not out yet. 
However, at that conference I get the opportuity to present how we started using actinia for Copernicus services in Norway.

Currently, I am planning my attendance and my question now is: will there be other actinia related activity at FOSS4G?
I would really appreciate to meet other members of the actinia community (users and developers) in Prizren and could reserve some time for discussions / chats.

Cheers
Stefan

**Comment by mmacata** [comment]
Hi Stefan,

unfortunately we from mundialis will not join this year :pensive: 
Will watch your recorded session then :)

Would be nice if anyone else interested in actinia will respond and you can meet.

Regards,
Carmen
